README.md

```diff
@@ -32,13 +32,37 @@ Mega will support code owners, which are a set of rules for defining who owns a
 
 For now, the entire open source community base on Git and GitHub. It's centralized model, and it's not suitable for growing speed of open source world. Mega is working on build a decentralized open source collaboration model with [ZTM](https://github.com/flomesh-io/ztm)(Zero Trust Model) and decentralized social network like [Nostr](https://nostr.com), [Matrix](https://matrix.org) and [Mastodon](https://joinmastodon.org).
 
-## Quick Start
-
-### Quick Try
-
-For now, we are developing on the macOS and Arch Linux. And quick start manuel in the [Quick start manuel to developing or testing](docs/development.md#quick-start-manuel-to-developing-or-testing).
-
-### Quick Review of Architecture
+## Quick Try Monorepo Engine with Docker
+
+For now, we are provide the run monorepo engine with docker, you can try it with the following steps:
```

Comment:
For now, the monorepo engine could be deployed on your host machine or insulated into containers.
For deploying through docker, follow the steps below:

wording